cmd/mysql/binlog_fetch.go

```diff
@@ -24,7 +27,7 @@ var binlogFetchCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		folder, err := internal.ConfigureFolder()
 		tracelog.ErrorLogger.FatalOnError(err)
-		mysql.HandleBinlogFetch(folder, fetchBackupName, fetchUntilTS)
+		mysql.HandleBinlogFetch(folder, fetchBackupName, fetchUntilTS, fetchUntilBinlogLastModifiedTS)
```

Comment:
what's the difference between fetchUntilTS and fetchUntilBinlogLastModifiedTS ?
can we use fetchUntilTs instead of fetchUntilBinlogLastModifiedTS

Reply:
Here I am introducing new  should distinguish two timestamps:
`fetchUntilTS` - is about time of individual transactions
`fetchUntilBinlogLastModifiedTS` - is about time when binlog was created

There are 2 properties:
`fetchUntilTS < fetchUntilBinlogLastModifiedTS` - because transactions dumped to binlog _before_ binlog upload
'--stop-datetime < fetchUntilBinlogLastModifiedTS' - it is new rule that fixes multi-host restores (check detailed explanation in another comment)